### PR TITLE
User creation

### LINKED ISSUE #357: Handoff to UAT / ISO
- Ensure test users are created for both UAT and ISO

### PR DESCRIPTION
**Pull request by sharonwarner**
Creates users for ISO, UAT, and internal testing

===

Resolves #357 

- [ ] Snapshots updated, if output HTML/JS has changed (`npm run test:update-snapshots`)
- [ ] Changes reviewed on mobile using devtools, if output HTML/CSS has changed
- [ ] Spanish checked by adding `?lng=es` to URL e.g. `/guide/benefits?lng=es`, if Spanish updated
- [ ] Automated tests added, if new functionality added
- [ ] Documentation (e.g. READMEs) updated, if relevant
- [ ] IE11 and Edge compatibility confirmed via manual testing in Browserstack, if new libraries added or newish JS/HTML/CSS features used for the first time in this codebase
- [ ] Accessibility & performance audit run using Google Lighthouse, if major changes made

### CODE REVIEW

File: scripts/create_test_users.py [hunk 1]
```diff
@@ -10,28 +10,61 @@
 SOURCE = 'script' + datetime.datetime.now().strftime('%Y%m%d.%H%M%S')
 
 NUM_WEEKS = 14
-PUA_FULL_TIME = 'PUA full-time'
-UI_FULL_TIME = 'UI full-time'
-UI_PART_TIME = 'UI part-time'
+PUA_FULL_TIME = 'PUA full time'
```
Comment (kalvinwang) [inline scripts/create_test_users.py:13]: good catch, I didn't notice this!

Comment (sharonwarner) [reply scripts/create_test_users.py:13]: I was so confused yesterday when none of the questions were showing up for the test users I created!

Comment (idealisms) [reply scripts/create_test_users.py:13]: Ah, I saw that too yesterday! Thanks for debugging and fixing.

**Comment by jdelloso** [comment]
For production smoke test, I wanted to give EDD 5 users from our internal team list so they can walk through every screen.

**Pull request merged**